src/components/ChoiceButton.js

```diff
@@ -4,11 +4,11 @@ import { useTranslation } from 'react-i18next';
 import PropTypes from 'prop-types';
 import { Button, Form } from 'semantic-ui-react';
 
-function Slope({ onChange, options, value }) {
+function ChoiceButton({ onChange, options, value, label }) {
   const { t } = useTranslation();
   return (
     <Form.Field>
-      <label>{t('slope.label')}</label>
+      <label>{t(`${label}.label`)}</label>
```

Comment:
Let's avoid magic like this. Better do the translation directly in the prop and only pass label through.